l10n_it_vat_settlement_date/views/account_move_views.xml

```diff
@@ -0,0 +1,34 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<!--
+  Copyright (c) 2021 Marco Colombo (https://github.com/TheMule71)
+  Copyright 2024 Simone Rubino - Aion Tech
+  License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+  -->
+<odoo>
+    <record id="view_move_form" model="ir.ui.view">
+        <field name="name">Add Settlement date to Journal Entry form view</field>
+        <field name="model">account.move</field>
+        <field name="inherit_id" ref="account.view_move_form" />
+        <field name="arch" type="xml">
+            <field name="invoice_date" position="after">
+                <field
+                    name="l10n_it_vat_settlement_date"
+                    attrs="{
+                        'invisible': [
+                            ('move_type', 'not in', (
+                                    'in_invoice',
+                                    'in_receipt',
```

Comment:
Perchè non `in_refund`?

Reply:
Perché in https://github.com/OCA/l10n-italy/issues/4034 è scritto
> Un ulteriore campo "data iva" in fattura fornitore

L'ho aggiunto anche alle ricevute fornitore perché mi sentivo generoso :smile:

Secondo te serve anche nelle note di credito fornitore?

Reply:
Abbondiamo ;)
Direi di sì, ad es. se un fornitore fa una nota di credito parziale su una fattura caricata in data competenza precedente, l'utente vorrà tenere la stessa data.

Reply:
:+1: